src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -300,18 +305,22 @@ function loadOperation(
         }
     }
 
+    const requestMethod = parseHttpRequestMethod(verb);
+    const generateConvenienceMethod = requestMethod !== RequestMethod.PATCH && (hasNoConvenienceApiDecorators || getExtensions(program, op).get(convenienceApiKey))
```

Comment:
@chunyu3 I have added handling for PATCH case and fixed test outputs. Please take a look.

Reply:
Thanks @AlexanderSher The change looks good for me.

Reply:
AFAIK - yes. We will revisit this when we full featured decorator for convenience methods